src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -34,14 +35,18 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
-        shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
+        shoppingCartService.addSession(movieSessionService.get(movieSessionId),
+                // -------------------comment for mentor----------------------
+                // do we need to handle exception here, or it`s ok use .get()?
+                userService.findByEmail(authentication.getName()).orElseThrow(
+                        () -> new RuntimeException("Couldn't add movie session to cart")));
```

Comment:
It is always nice to throw with a message then just getting the value, `get()` is usually used when it is guaranteed that we will have a value there, our situation looks like it is, but you never know, so let's leave it as it is 😉

Reply:
Also, add id's to the exception message